hera_sim/sigchain.py

```diff
@@ -488,6 +499,95 @@ def __call__(self, freqs, autovis, **kwargs):
         return crosstalk_spectrum
 
 
+class OverAirCrossCoupling(Crosstalk):
+    """Crosstalk based on HERA Memo on H1C crosstalk."""
+
+    def __init__(
+        self,
+        emitter_pos=0,
+        cable_delays=None,
+        base_amp=2e-5,
+        amp_slope=-1,
+        amp_decay_base=10,
+        Ncopies=10,
+        amp_jitter=0,
+        dly_jitter=0,
+        max_delay=2000,
+        amp_decay_fac=1e-2,
+    ):
+        super().__init__(
+            emitter_pos=emitter_pos,
+            cable_delays=cable_delays or {},
+            base_amp=base_amp,
+            amp_slope=amp_slope,
+            amp_decay_base=amp_decay_base,
+            Ncopies=Ncopies,
+            amp_jitter=amp_jitter,
+            dly_jitter=dly_jitter,
+            max_delay=max_delay,
+            amp_decay_fac=amp_decay_fac,
+        )
+
+    def __call__(
+        self,
+        freqs,
+        antpair,
+        antpos,
+        autovis_i,
+        autovis_j,
+        **kwargs,
+    ):
+        """Actually simulate the crosstalk."""
+        self._check_kwargs(**kwargs)
+        (
+            emitter_pos,
+            cable_delay,
+            base_amp,
+            amp_slope,
+            amp_decay_base,
+            Ncopies,
+            amp_jitter,
+            dly_jitter,
+            max_delay,
+            amp_decay_fac,
+        ) = self._extract_kwarg_values(**kwargs)
+
+        ai, aj = antpair
+        xi = antpos[ai]
+        xj = antpos[aj]
+
+        def log(x):
+            return np.log(x) / np.log(amp_decay_base)
```

Comment:
not sure if it matters, but if  `amp_decay_base` is constant, this would be faster if it was evaluated outside the function.

Reply:
`amp_decay_base` is indeed constant within the scope of the `__call__` method, so I can clean this up a bit. Thanks for the tip!